DemoInfo/Events.cs

```diff
@@ -405,7 +418,7 @@ public static EquipmentElement MapEquipment(string OriginalString)
 			if (OriginalString.Contains ("knife") || OriginalString == "bayonet") {
 				weapon = EquipmentElement.Knife;
 			}
-
+			
```

Comment:
You have unneeded spaces here. It's a bit weird for us to complain about formatting, but yeah :P